lib/configParser.js

```diff
@@ -58,9 +90,8 @@ var ConfigParser = function() {
  */
 var merge_ = function(into, from) {
   for (var key in from) {
-    if (into[key] instanceof Object &&
-        !(into[key] instanceof Array) &&
-        !(into[key] instanceof Function)) {
+    if ((isArrayLike_(into[key]) && isArrayLike_(from[key])) || 
+        (isDictionary_(into[key]) && isDictionary_(from[key]))) {
```

Comment:
This means that arrays won't be replaced anymore, right? So it would change behavior where you have a config file with `specs: ['test1.js', 'test2.js']` and then use the command line with `--specs test1.js` so that it would not override and just run test1.js. 

I like that this enables multiConf from the command line, but I'm not sure it's worth losing the scenario above.